lib/Twig/Extension/Core.php

```diff
@@ -852,6 +852,34 @@ function twig_in_filter($value, $compare)
     return false;
 }
 
+/** 
+ * Returns a trimmed string.
+ * 
+ * @param string $string
+ * @param string $characterMask
+ * @param string $mode
+ * @return string
+ */
+function twig_trim_filter($string, $characterMask = null, $mode = 'both')
```

Comment:
If I'm right, in English they say *"trim from [left/right/both sides]"* ... so maybe we could rename the `$mode` parameter to either `from` or `side`:

```twig
{{ '...'|trim(mode='both') }}
{{ '...'|trim(mode='left') }}
{{ '...'|trim(mode='right') }}

{{ '...'|trim(from='both') }}
{{ '...'|trim(from='left') }}
{{ '...'|trim(from='right') }}

{{ '...'|trim(side='both') }}
{{ '...'|trim(side='left') }}
{{ '...'|trim(side='right') }}
```